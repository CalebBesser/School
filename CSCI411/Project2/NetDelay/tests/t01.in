11
addEdge 0 3 11
addEdge 1 2 13
addEdge 2 3 8
addEdge 3 10 8
addEdge 4 6 12
addEdge 5 6 16
addEdge 6 9 14
addEdge 7 8 35
addEdge 7 9 20
addEdge 7 10 9
printGraphW
findNetDelay
