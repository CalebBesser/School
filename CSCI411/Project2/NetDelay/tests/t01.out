Graph is:
0: 3/11	
1: 2/13	
2: 1/13	3/8	
3: 0/11	2/8	10/8	
4: 6/12	
5: 6/16	
6: 4/12	5/16	9/14	
7: 8/35	9/20	10/9	
8: 7/35	
9: 6/14	7/20	
10: 3/8	7/9	

The longest path is:
1 2 3 10 7 9 6 5 
The length of the longest path is 88.
